Modules/Core/Transform/include/itkMatrixOffsetTransformBase.h

```diff
@@ -507,7 +507,10 @@ class ITK_TEMPLATE_EXPORT MatrixOffsetTransformBase
    * to values specified by the caller.  If the arguments are
    * omitted, then the MatrixOffsetTransformBase is initialized to an identity
    * transformation in the appropriate number of dimensions. */
-  MatrixOffsetTransformBase(const MatrixType & matrix, const OutputVectorType & offset);
+#if !defined(ITK_LEGACY_REMOVE)
+  [[deprecated("Removed unsued constructor")]] MatrixOffsetTransformBase(const MatrixType &       matrix,
```

Comment:
Can you please fix this little typo here, "unsued"? Although I know it's not _that_ bad, we won't get _sued_  😸

Reply:
Thanks for fixing the typo 👍